HelpSource/Classes/Splay.schelp

```diff
@@ -70,12 +83,36 @@ x = { |spread = 1, level = 0.2, center = 0.0|
 }.play;
 )
 
+::
+NOTE:: levelComp new takes multiple input options:
+true, false as before for rough equal power level compensation,
+and newly added: a float between 0 and 1. This is to offer more choices for level compensation, such as equal maximum amplitude.
+::
 
-// with mouse control
-(
-x = { var src;
-	src = SinOsc.ar({ |i| LFNoise2.kr(rrand(10, 20), 200, i + 3 * 100) } ! 10);
-	Splay.ar(src, MouseY.kr(1, 0), 0.2, MouseX.kr(-1, 1));
-}.play;
-)
+For a full discussion, see link::Classes/LevelComp::;
+here is an overview of the levelComp variants:
+
+code::
+// default is equal power comp: level / ( numchans.sqrt)
```

Comment:
In a number of places in this code block, there are random single spaces within parentheses on one side or the other. The code style guide prefers not spaces, e.g. `level / (numchans.sqrt)`

Reply:
done

Reply:
Not sure if this is a github thing, but the changes don't appear here for me.
I see the paren issue on lines 103, 104, 121, 122

Reply:
ok, got them now